config/routes.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  #!original should uncomment the links below...
+  # resources :users, only: [:show]
+
+  #! Defines test routes for users actions... delete after exercise
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:id/posts/:post_id', to: 'posts#show', as: 'user_post'
+  get '/users', to:'users#index', as: 'users'
+  get '/users/:id', to: 'users#show', as: 'user'
```

Comment:
- [ ] @admirerbrown kindly check on your `id` it should be `user_id` as shown in the project requirements, kindly correct the mistake.
![image](https://user-images.githubusercontent.com/61408860/229561327-59ef0e07-52f7-49c2-b8e3-f512c2730f2a.png)